src/nvim/terminal.c

```diff
@@ -500,9 +500,6 @@ void terminal_open(Terminal **termpp, buf_T *buf, TerminalOptions opts)
   // events from this queue are copied back onto the main event queue.
   term->pending.events = multiqueue_new(NULL, NULL);
 
-  aco_save_T aco;
-  aucmd_prepbuf(&aco, buf);
-
   refresh_screen(term, buf);
   set_option_value(kOptBuftype, STATIC_CSTR_AS_OPTVAL("terminal"), OPT_LOCAL);
```

Comment:
code here assumes curbuf, and code below assumes curwin, so might not be correct to remove the prepbuf here